tests/Guzzle5/phpstan.neon

```diff
@@ -6,5 +6,12 @@ parameters:
 	bootstrapFiles:
 		- vendor/autoload.php
 	reportUnmatchedIgnoredErrors: false
+	ignoreErrors:
+		-
+			message: '#^PHPDoc tag \@throws with type Psr\\Http\\Client\\ClientExceptionInterface is not subtype of Throwable$#'
+			path: ../../src/Interfaces/HttpClientInterface.php
```

Comment:
The namespace `Psr\Http\Client` is unknown. Maybe is would be interesting to import ClientExceptionInterface in the codebase as well?

Reply:
I'm troubled because it was my first idea to import this Interface but I had an error with phpstan-g5 related to the Throwable class (not available in php5).
But it seems OK now, no errors ! I added it.